test/functional/developer_portal/admin/account/personal_details_controller_test.rb

```diff
@@ -65,12 +65,17 @@ def setup
   end
 
   test 'changing password is audited' do
-    login_as @buyer.admins.first
+    user = @buyer.admins.first
+    login_as user
+
     assert_difference(Audited.audit_class.method(:count)) do
       User.with_synchronous_auditing do
         put :update, params: { user: {current_password: 'supersecret', password: 'new_password', password_confirmation: 'new_password'} }
       end
     end
+
+    expected = [Audited::Auditor::AuditedInstanceMethods::REDACTED] * 2
+    assert_equal expected,user.audits.last.audited_changes['password_digest']
```

Comment:
I don't understand why it is `*2` but better than the actual value :)

Reply:
```
[Audited::Auditor::AuditedInstanceMethods::REDACTED] * 2
```

is a cool way to do

```
[Audited::Auditor::AuditedInstanceMethods::REDACTED, Audited::Auditor::AuditedInstanceMethods::REDACTED]
```

Reply:
ah, now I saw the `[]`